docs/node_sdk_release.md

```diff
@@ -0,0 +1,36 @@
+# Node SDK Release Process Document
+
+Using v1.4 branch and last release of v1.4.15 as an example throughout.
+
+## Pre-Check
+
+Check top level `package.json` file properties **version** and **tag** are one above the current release and correct for publishing snapshots.  Expect *v1.4.16-snapshot* and *unstable-1.4*.  If this is not the case then builds are likely to fail.  Consider fixing and creating a PR or move straight onto submit a PR for a new release.
+
+## Submit Release PR
+
+- Create a jira for publish  - publish v1.4.16.
+- Run `scripts/changelog.sh` script from the project root directory.  It takes two arguments; last release commit hash and next release version number - `./scripts/changelog.sh <commithash> 1.4.16`.
+- Update the **version** and **tag** properties in the top-level 'package.json' file to be the appropriate values - next release version *v1.4.16* and release npm tag *latest-1.4*.
```

Comment:
You might have meant to use back-ticks around `package.json` instead of apostrophes: 'package.json'

Reply:
Next release version would be `1.4.16`. No `v` at the start.

Reply:
Changed